src/DiffTests.jl

```diff
@@ -25,15 +25,15 @@ names, one should not expect these functions to be "correct" for their original
 num2num_1(x) = sin(x)^2 / cos(x)^2
 num2num_2(x) = 2*x + sqrt(x*x*x)
 num2num_3(x) = 10.31^(x + x) - x
-num2num_4(x) = 1
+num2num_4(x) = one(x)
```

Comment:
I assume that for num2num functions the contract is to return the same type as the input, so it could be tested in the unit tests along with type stability tests. It could be reverted, along with the tests.

Reply:
I reverted it